pylint.cfg

```diff
@@ -1,4 +1,4 @@
 [MESSAGES CONTROL]
-disable=C0330,I1101,R0801,R0902,R0903,R0913,R0914,W1202,W1203,W1510
+disable=C0330,I1101,E1101,R0801,R0902,R0903,R0913,R0914,W1202,W1203,W1510
```

Comment:
Hmmm, not quite sure where there's something dynamic happening in the changed code in terms of class member access. The Pydantic models are defined and everything seems to be typed. Does this show up in multiple places? If only in one or few, better to use `# pylint: disable=E1101` on the offending lines. I suspect that this can be avoided altogether.